index.html

```diff
@@ -420,6 +439,28 @@
         <!--<img id="austria" src="icons/countries/austria.svg" class="imageCache"></img>-->
         <!--<img id="poland" src="icons/countries/poland.svg" class="imageCache"></img>-->
     </div>
+
+    <div id="myModal" class="modal">
+        <div class="modal-content">
+            <div class="messagePart">
+                <img class="close" onclick="closeVoiceMessagePopapp()" src="icons/voiceMessage/page_1.png"/>
+                <img class="audioImg" src="icons/voiceMessage/group_5.png"/>
+               <div class="audio">
+                    <audio id="audioSRC" src="icons/voiceMessage/file_example_MP3_700KB.mp3" ontimeupdate="updateAudioMessageTime()"></audio>
+                    <span id="audioTime" class="audioTime">00:00</span>
+                    <img id="audioTrack" class="audioTrack" src="icons/voiceMessage/rectangle.png"/>
+                    <img id="trackerPoint" class="trackerPoint" src="icons/voiceMessage/blue.png"/>
+                    <img class="audioPlay" onclick="playBTN()" src="icons/voiceMessage/triangle.png"/>
+                </div>
+                <div class="border"></div>
+                <p class="wroteMessage">שלום, כאן א', מפקד טייסת אבירי הזנב הכתום. אנו אנשי הטייסת, הטכנאים והלוחמים בסדיר ובמילואים, מבקשים לחזק את ידם של הצוותים הרפואיים וכלל האזרחים בימים לא פשוטים אלו. מאחלים לכולנו חג עצמאות שמח! </p>
```

Comment:
This text should be dynamic, using information from json files.

Reply:
@mayPeleg

Reply:
@mayPeleg Please, add to the json a "message" field, and use it dynamically.

Reply:
Done